server/src/internalClusterTest/java/org/opensearch/search/pit/DeletePitMultiNodeIT.java

```diff
@@ -52,7 +59,23 @@
  * Multi node integration tests for delete PIT use cases
  */
 @OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.SUITE, numDataNodes = 2)
-public class DeletePitMultiNodeIT extends OpenSearchIntegTestCase {
+public class DeletePitMultiNodeIT extends ParameterizedOpenSearchIntegTestCase {
```

Comment:
This class doesn't need parameterization as it is verifying deletion of Pits only